lbrynet/lbrynet_daemon/auth/client.py

```diff
@@ -46,7 +46,7 @@ def call(self, method, params={}):
         pre_auth_post_data = {
             'version': '1.1',
             'method': method,
-            'params': [params],
+            'params': params,
```

Comment:
The change itself was done in a previous commit. This is just to get our client using the new way.